test/timeseries_path_partitioner/processor-spec.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const { DataEntity } = require('@terascope/utils');
+const { WorkerTestHarness } = require('teraslice-test-harness');
+
+describe('Timeseries path partitioner', () => {
+    let harness;
+
+    const data = [
+        DataEntity.make(
+            {
+                date: '2020-01-17T19:21:52.159Z',
+                text: 'test data'
+            }
+        )
+    ];
+
+    beforeAll(async () => {
```

Comment:
This beforeAll isn’t needed because it only applies to one test and this worker harness instance should be created in your `it` block (like the other ones)

Reply:
Fixed